### ISSUE TITLE
reword "EP Lib" and "EP Lab" in header?

### ISSUE THREAD

**Issue by craigberry** [earlyprint/earlyprint.github.io#19]
Can we simply say "Library" and "Laboratory" in the header?  It would only add 3 more characters in toto and for me would be much clearer, especially when one is deep into one of the various tools rather than looking at the top-level page. I didn't know what EP Lib and EP Lab meant the first time I saw them and I already know a fair amount about the things they point to.  "EP" is redundant since everything here is related to EarlyPrint.  "Lab" is a common enough abbreviation for laboratory, so people would probably guess the link doesn't take them to a site about Labrador retrievers, but "Lib" is not a known abbreviation for library, at least outside of techie circles.

**Comment by jfloewen** [comment]
I like this idea, but wiser heads should prevail.

On Nov 3, 2019, at 8:00 AM, Craig A. Berry <notifications@github.com<mailto:notifications@github.com>> wrote:


Can we simply say "Library" and "Laboratory" in the header? It would only add 3 more characters in toto and for me would be much clearer, especially when one is deep into one of the various tools rather than looking at the top-level page. I didn't know what EP Lib and EP Lab meant the first time I saw them and I already know a fair amount about the things they point to. "EP" is redundant since everything here is related to EarlyPrint. "Lab" is a common enough abbreviation for laboratory, so people would probably guess the link doesn't take them to a site about Labrador retrievers, but "Lib" is not a known abbreviation for library, at least outside of techie circles.

**Comment by jrladd** [comment]
This sounds good to me. I think it should be ”Library” and “Lab,” since
“Laboratory” is a bit wordy and everyone knows what Lab stands for. We
already made a similar change to the URLs along the same lines (
earlyprint.org/lab instead of earlyprint.org/eplab).

In fact, this raises a couple questions about consistency. Should we be
using library.earlyprint.org instead of texts.earlyprint.org? And, should
we be using subdomains or directories across the board for the top level
sites? So either “earlyprint.org/lab” and “earlyprint.org/library” OR “
lab.earlyprint.org” and “library.earlyprint.org”? I would prefer the first
option (directories), since a subdomain will break the Jekyll organizing
scheme for the Lab tools.

**Comment by craigberry** [comment]
As far as I know, you can't put directories in DNS, so there would be no way to specify that the different sites run from different servers.  I have no objection to putting the Library at library.earlyprint.org, but we've been using texts.earlyprint.org for a couple of years and have announced it in various places, so we should keep it in DNS and pointing to the same place for the foreseeable future.  Which probably means maintaining certificates for both subdomains.

**Comment by jrladd** [comment]
Okay, this is a good reason for sticking with a directory for the lab and a
subdomain for the library. I don’t regard changing to library.earlyprint.org
as essential, and I agree that we should retain texts.earlyprint to avoid
breaking links. Do others feel its worth going through the work of changing
over? Or should we leave it as is?

**Comment by spenteco** [comment]
I like the idea of leaving things as they are now (the lab at earlyprint.org/lab) . . .

On 11/3/19 9:47 AM, JR Ladd wrote:
Okay, this is a good reason for sticking with a directory for the lab and a
subdomain for the library. I don’t regard changing to library.earlyprint.org
as essential, and I agree that we should retain texts.earlyprint to avoid
breaking links. Do others feel its worth going through the work of changing
over? Or should we leave it as is?

**Comment by jrladd** [comment]
I'm glad we had the conversation about URLs, but I agree that it's not worth changing at the moment. I've changed "EP Lib" to "Library" and "EP Lab" to "Lab" in the site header.
